addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorMotionHandler.java

```diff
@@ -41,6 +41,12 @@ public XiaomiSensorMotionHandler(Thing thing) {
     @Override
     void parseReport(JsonObject data) {
```

Comment:
Just a small bug bear of mine...can you define the strings ("status", "motion", "lux") as string constants? Makes maintenance in the future easier.

Reply:
Done, thank you!